index.html

```diff
@@ -85,5 +85,29 @@
     of code. If it needs to be a bit bigger, it will be.
   </li> 
 </ol> 
+
+<p>
+In addition to these stated goals, we try our best to uphold the following
+values when considering contributions to the project:
+</p>
+
+<ul>
+  <li>
+    Never break configuration files or existing workflows. Breaking changes
+    require a major version bump (v4 → v5).
```

Comment:
Our model largely is semantic versioning, but strictly speaking, http://semver.org (the authority on the term, IIUC) only applies to public APIs, i.e. it targets libraries, not software for end users.

I think the point as it is currently written conveys that breaking changes will come with a major version bump, and I don’t think people actually care enough about what a patch update (e.g. 4.13.1) really means for us to come up with specific rules about it :)

The binding changes were not deliberately incompatible; we (at least I) only realized this issue after the release was already done. Given that unpublishing a release seems like a confusing move to our users, I wonder what a good position on accidentally incompatible releases would be.